pkg/operator/ceph/object/rgw-probe.sh

```diff
@@ -14,7 +15,7 @@ PROBE_ERR_CODE=124
 STARTUP_TYPE='startup'
 READINESS_TYPE='readiness'
 
-RGW_URL="$PROBE_PROTOCOL://0.0.0.0:$PROBE_PORT$PROBE_PATH"
+RGW_URL="$PROBE_PROTOCOL://$PROBE_HOST:$PROBE_PORT$PROBE_PATH"
 
```

Comment:
I'm very concerned about the possibility of breaking the probe. What are the configurations where you have tested this?

Reply:
I'm also curious whether this change is needed. Not having to worry about IPs in the probe would be best, though I know it might be required.

Did you try leaving this at 0.0.0.0 when using host networking with the other changes to see if the probe still gets RGW health info?